### DISCUSSION TITLE
Does Vorta run in the background

### DISCUSSION THREAD

**Discussion by QuarkZ26** [borgbase/vorta#1280] [General]
Hello, 

Does Vorta run in the background? I see options to start at login and display background exit dialog, but none of those work for me, so I'm wondering if it's just running in the background.

**Comment by m3nu** [comment]
Yes, usually it runs int the BG and stays accessible via tray menu. The ‘Start at login’ option is implemented for macOS and some Linux flavors, but maybe not all. What desktop are you on?

**Reply by QuarkZ26** [reply]
I'm on Kubuntu 22.04. I sure can make it start another way, I'm just curious whether it has a service of some type or if it closes when I quit (I'm assuming the background dialog also isn't supported since I don't get any)

**Reply by real-yfprojects** [reply]
It doesn't use a real service but simply hides the GUI without stopping the actual process.
The background dialog is only shown if their is no tray icon available.
